src/signature.c

```diff
@@ -37,7 +37,7 @@
 #include "swupd.h"
 
 #define BUFFER_SIZE   4096
-
+#define CERTNAME "/usr/share/clear/update-ca/ClearLinuxRoot.pem"
```

Comment:
I would love to see this value set by a configure option instead of hardcoding the path.